src/main/java/uk/gov/hmcts/divorce/payment/model/Payment.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.hmcts.divorce.payment.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.ccd.sdk.api.CCD;
+import uk.gov.hmcts.ccd.sdk.type.MoneyGBP;
+import uk.gov.hmcts.divorce.common.model.access.DefaultAccess;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+public class Payment {
+
+    @CCD(
+        label = "Payment Date",
+        access = {DefaultAccess.class}
```

Comment:
Do fields on a complex type have access control or is it only the complex type as a whole?